tests/upgrades/test_yum_plugins.py

```diff
@@ -78,7 +76,7 @@ def _check_yum_plugins_count(self, client_container_id):
     def _create_custom_tools_repos(self, product):
         """Create custom tools repo and sync it"""
 
-        tools_repo_url = settings.repos.sattools_repo[DISTRO_RHEL7]
+        tools_repo_url = settings.repos.sattools_repo['rhel7']
         if None in [tools_repo_url]:
             raise ValueError(f'The Tools Repo URL {self.client_os} is not provided!')
```

Comment:
```suggestion
            raise ValueError(f'The Tools Repo URL rhel7 is not provided!')
```

Reply:
That could very well be the right call. However, I cannot determine the origin of `self.client_os`. @ColeHiggins2 do you know?

Reply:
yeah .... `self` is the test class, and I don't see this getting set anywhere. I think I'd go with @shweta83 suggestion here since `rhel7` is hardcoded in the key lookup relevant to this statement.

Reply:
Since this line isn't actually touched by the PR, I'm going to merge as-is. The test owner can address (or @shweta83 you can open a one-liner!) this `client_os` reference.